src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -295,13 +335,25 @@ public void onTraversalStart() {
         final LinkedHashMap<String, String> unaccountedForOverrideAnnotations = getUnaccountedForAnnotations( dataSourceFactories, annotationOverridesMap );
 
         // Set up our output renderer:
-        // TODO: in the future this should be encapsulated into a factory for output renderers based on an input argument.
-        outputRenderer = new VcfOutputRenderer(getHeaderForVariants(),
-                                               createVCFWriter(outputFile),
-                                               dataSourceFactories,
-                                               unaccountedForDefaultAnnotations,
-                                               unaccountedForOverrideAnnotations);
-
+        switch (outputFormatType) {
+            case MAF:
+                outputRenderer = new MafOutputRenderer(outputFile.toPath(),
+                        dataSourceFactories,
+                        unaccountedForDefaultAnnotations,
+                        unaccountedForOverrideAnnotations,
+                        getHeaderForVariants(),
+                        getDefaultToolVCFHeaderLines().stream().map(x -> x.toString()).collect(Collectors.toCollection(LinkedHashSet::new)));
```

Comment:
A more natural way to write `x -> x.toString()` is `Object::toString`

Reply:
Sounds good.  Fixed!